app/assets/scss/_search-page.scss

```diff
@@ -1,92 +0,0 @@
-@import "search/_search-summary.scss";
-
-.search-summary {
```

Comment:
We also have `p.instruction-list-item-top.search-summary` in `instruction-list.scss`, which can probably go.

Reply:
Also it's interesting, because this class is used by `templates/toolkit/search-summary.html`, which we aren't removing (and is brought in by the toolkit). So now we have a template that our toolkit brings in but doesn't have any styling with it. Not sure what we think about this.

Reply:
Yeah I'm not 100% sure on it either. I assume the template is being brought in by gulp which is copying all the templates?

It feels like there are three options
- change gulp to only bring in templates we need whilst also only bringing in the related scss
- only bring in scss we need but bring in all the templates
- bring in everything from both

All of them have pros and cons so I'm not entirely sure which I think is best yet. Any ideas?

Reply:
It feels pretty weird to have half of a component.

Reply:
I'm not sure what to do with this one. Should I put ```.search-summary back``` ?

Reply:
Not sure. I think maybe we need to have a chat around it. It's probably clearer for us doing this work if we remove it, as that's the principle we've applied everywhere else (ie just remove everything that isn't directly used by the app). We'll need to really clearly document and communicate our decisions around this (and the wider piece of work) though.

Reply:
Good idea. Let's talk about this issue tomorrow in person tomorrow. We can also talk about how we are making sure our decisions and new principles are documented and communicated.

Reply:
@samuelhwilliams can I remove this file then?

Reply:
Did we reach a consensus on this @samuelhwilliams @MuriloDalRi or shall we speak about it in person to make sure we are all on the same page? BTW this was the thing I was trying to bring up yesterday morning but forgot the details

Reply:
Decided to not import the scss files we don't need regardless of if the related html templates are bulk imported by gulp.